bundle/jsky.app.ot/src/main/java/jsky/app/ot/scilib/ScienceLibraryInfo.java

```diff
@@ -45,6 +46,7 @@ final class ScienceLibraryInfo {
         _libInfo.put(InstNIFS.SP_TYPE, new ScienceLibraryInfo(Site.GN, "GN-NIFS-library"));
         _libInfo.put(InstNIRI.SP_TYPE, new ScienceLibraryInfo(Site.GN, "GN-NIRI-library"));
         _libInfo.put(InstTexes.SP_TYPE, new ScienceLibraryInfo(Site.GN, "GN-TEXES-library"));
+        _libInfo.put(Igrins2.SP_TYPE, new ScienceLibraryInfo(Site.GN, "GN-IGRINS2-library"));
```

Comment:
Does the program already exist in the ODB?

Reply:
It does, I tested it importing one but science confirmed it exists at GN odb